api/swagger.yaml

```diff
@@ -4187,11 +4187,11 @@ paths:
         - name: "path"
           in: "query"
           required: true
-          description: "Path to a directory in the container to extract the archive’s contents into. "
+          description: "Path to a directory in the container to extract the archive's contents into. "
           type: "string"
         - name: "noOverwriteDirNonDir"
           in: "query"
-          description: "If “1”, “true”, or “True” then it will be an error if unpacking the given content would cause an existing directory to be replaced with a non-directory and vice versa."
+          description: "If `1`, `true`, or `True` then it will be an error if unpacking the given content would cause an existing directory to be replaced with a non-directory and vice versa."
```

Comment:
![image](https://cloud.githubusercontent.com/assets/19368056/21354359/b650a6bc-c704-11e6-8d1a-d8dcda64c737.png)
I think there should be corresponding with this.